main/modes/platformer/entity.c

```diff
@@ -138,16 +138,16 @@ void updatePlayer(entity_t *self)
 {
     if (self->gameData->btnState & BTN_B)
     {
-        self->xMaxSpeed = 132;
+        self->xMaxSpeed = 52;
     }
     else
     {
-        self->xMaxSpeed = 72;
+        self->xMaxSpeed = 30;
     }
 
     if (self->gameData->btnState & LEFT)
     {
-        self->xspeed -= (self->falling) ? 12 : 16;
+        self->xspeed -= (self->falling && self->xspeed < 0) ? (self->xspeed < -24) ? 0 : 2 : 3;
```

Comment:
I find nested ternary operators with logical operators generally hard to read. One line of code is tight, but may be hard to grok next year. Think about if/else next time